src/conv-core/cpuaffinity.C

```diff
@@ -37,16 +37,12 @@ CpvDeclare(void *, myProcStatFP);
 
 CmiHwlocTopology CmiHwlocTopologyLocal;
 
+static hwloc_topology_t topology, legacy_topology;
```

Comment:
A comment here explaining the difference between these two might be good

Reply:
Done